tailwind.config.js

```diff
@@ -15,6 +15,56 @@ module.exports = {
           800: '#08072e',
         },
       },
+      typography: ({ theme }) => ({
+        DEFAULT: {
+          css: {
+            '--tw-prose-body': theme('colors.brand.100'),
+            '--tw-prose-headings': theme('colors.brand.100'),
+            '--tw-prose-bold': theme('colors.brand.100'),
+            '--tw-prose-links': theme('colors.brand.300'),
+            '--tw-prose-quotes': theme('colors.brand.100'),
+            '--tw-prose-quote-borders': theme('colors.brand.300'),
+            '--tw-prose-counters': theme('colors.brand.100'),
+            '--tw-prose-bullets': theme('colors.brand.100'),
+            '--tw-prose-code': theme('colors.brand.100'),
+            '--tw-prose-pre-bg': theme('colors.brand.700'),
+            '--tw-prose-pre-code': theme('colors.brand.300'),
+
+            a: {
+              textDecoration: 'none',
+              '&:hover': {
+                textDecoration: 'underline',
+              },
+            },
+
+            code: {
+              '&::before, &::after': {
+                content: 'none !important',
+              },
+            },
```

Comment:
What does this do?

Reply:
This removes the visible backticks from the inline code style.

**default**
![CleanShot 2024-05-02 at 10 56 37](https://github.com/FilecoinFoundationWeb/filecoin-foundation/assets/20277959/10dcfd04-9981-4d5f-97aa-a5e528a6f267)

**now**
![CleanShot 2024-05-02 at 10 57 08](https://github.com/FilecoinFoundationWeb/filecoin-foundation/assets/20277959/d34299bb-ec42-4b29-b48b-79a7820d6cc8)